Update Fleet user guide docs for 7.14 (#935)
elastic/observability-docs: docs/en/ingest-management/getting-started.asciidoc

contain any number of integrations for collecting observability data from the various
services running on your host.

> . In {kib}, go to **Management > Integrations**.
> Use the search bar to find the Elastic Synthetics integration.

Reviewer: ```suggestion
. In {kib}, go to **Management > Integrations**,
and search for the **Elastic Synthetics** integration.
```

NOTE: For these steps, we assume that you have `nginx` running on some of your
infrastructure, and want to collect logs and metrics from it.

> . In {kib}, go back to **Management > Integrations**,
> and use the search bar to find the Nginx integration.

Reviewer: ```suggestion
. In {kib}, go back to **Management > Integrations**,
and search for the **Nginx** integration.
```
